pkg/service/backup/worker_schema.go

```diff
@@ -72,6 +72,13 @@ func (w *worker) safeBackupAndRestoreSchemaDump(ctx context.Context, descSchemaH
 	return nil
 }
 
+func (w *worker) raftReadBarier(ctx context.Context, session gocqlx.Session, useReadBarierAPI bool) error {
```

Comment:
Why do you prefer to have `useReadBarierAPI` as a `bool` here?
It's nice that you created a dedicated type for raft read barrier methods, so why don't we use it here?

Reply:
Thought it might be overkill here... but yeah, maybe it's better to use it here as well for the sake of consistency